flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-elasticsearch/src/test/java/org/apache/flink/cdc/connectors/elasticsearch/sink/Elasticsearch6DataSinkITCaseTest.java

```diff
@@ -0,0 +1,390 @@
+            long expectedTimestamp)
+            throws Exception {
+        GetRequest getRequest = new GetRequest(tableId.toString()).id(id);
+
+        GetResponse response = client.get(getRequest, RequestOptions.DEFAULT);
+
+        LOG.debug("Response source: {}", response.getSource());
+
+        assertThat(response.getSource()).isNotNull();
+        assertThat(((Number) response.getSource().get("id")).intValue()).isEqualTo(expectedId);
+        assertThat(((Number) response.getSource().get("number")).doubleValue())
+                .isEqualTo(expectedNumber);
+        assertThat(response.getSource().get("name")).isEqualTo(expectedName);
+        assertThat(response.getSource().get("bool")).isEqualTo(expectedBool);
+        assertThat(((Number) response.getSource().get("tinyint")).byteValue())
+                .isEqualTo(expectedTinyint);
+        assertThat(((Number) response.getSource().get("smallint")).shortValue())
+                .isEqualTo(expectedSmallint);
+        assertThat(((Number) response.getSource().get("bigint")).longValue())
+                .isEqualTo(expectedBigint);
+        assertThat(((Number) response.getSource().get("float")).floatValue())
+                .isEqualTo(expectedFloat);
+        assertThat(new BigDecimal(response.getSource().get("decimal").toString()))
+                .isEqualTo(expectedDecimal);
+
+        String timestampString = response.getSource().get("timestamp").toString();
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss.SSSSSS");
+        LocalDateTime dateTime = LocalDateTime.parse(timestampString, formatter);
+        long timestampMillis = dateTime.toInstant(ZoneOffset.UTC).toEpochMilli();
+        assertThat(timestampMillis).isEqualTo(expectedTimestamp);
+    }
+
+    private void verifyDeletedData(TableId tableId, String id) throws Exception {
+        GetRequest getRequest = new GetRequest(tableId.toString()).id(id);
+        GetResponse response = client.get(getRequest, RequestOptions.DEFAULT);
+
+        assertThat(response.isExists()).isFalse();
+    }
+
+    private void verifyInsertedDataWithNewColumn(
+            TableId tableId,
+            String id,
+            int expectedId,
+            double expectedNumber,
+            String expectedName,
+            boolean expectedExtraBool)
+            throws Exception {
+        GetRequest getRequest = new GetRequest(tableId.toString()).id(id);
+        GetResponse response = client.get(getRequest, RequestOptions.DEFAULT);
+
+        assertThat(response.getSource()).isNotNull();
+        assertThat(response.getSource().get("id")).isEqualTo(expectedId);
+        assertThat(response.getSource().get("number")).isEqualTo(expectedNumber);
+        assertThat(response.getSource().get("name")).isEqualTo(expectedName);
+        assertThat(response.getSource().get("extra_bool")).isEqualTo(expectedExtraBool);
+    }
+
+    private List<Event> createTestEvents(TableId tableId) {
```

Comment:
Looks like `createTestEvents`/`createTestEventsWithDelete`/`createTestEventsWithAddColumn` are common method in `Elasticsearch6DataSinkITCaseTest`/`Elasticsearch7DataSinkITCaseTest`/`ElasticsearchDataSinkITCaseTest`, we can extract them to an util class of a parent class.